.gitignore

```diff
@@ -1 +1,4 @@
 /coverage.txt
+/vendor
```

Comment:
should we remove that lines from .gitignore. go-billy is not using dep right now.

Reply:
that's why I added ;) to avoid a mistake and commit vendor dir